### ISSUE TITLE
ActionServer don't include the SpeedSlider in duration time of the movement

### ISSUE THREAD

**Issue by AlcatrAAz** [fzi-forschungszentrum-informatik/cartesian_ros_control#21]
Hello

I found a little bug in the Action Server when you don't have the SpeedSlider (at the Teachpanel) at 100%.

100%, Move Duration 5s:
-> The robot moves 5s to the target position
-> I get for 5 seconds a feedback from the Action Server

50%, Move Duration 5s:
-> The robot moves 10s to the target position
-> I get for 5 seconds a feedback from the Action Server

So the Action Server don't include the SpeedSlider

**Comment by stefanscherzinger** [comment]
Thanks @AlcatrAAz for reporting this.

**Comment by christianlandgraf** [comment]
I'm not sure, if this is related to the problem, but I am getting the following error (or rather info) when loading the cartesian trajectory controller with an UR10e:

![grafik](https://user-images.githubusercontent.com/28567122/120351088-d9b68a80-c2ff-11eb-8150-4ca54e765c70.png)

I am working on the beta-testing branch of the Universal ROS Driver. Do I need to change something or is the feature indeed missing?

Thanks for the help!

**Comment by stefanscherzinger** [comment]
@christianlandgraf Yes, that's related to what @AlcatrAAz  reported (in part). Unfortunately, losing this feature was an unwanted consequence of 3462444 and has been missing since. We are currently working on this (along with some further enhancements to the pass_through_controllers) and it should be back soon.

**Comment by fmauch** [comment]
This should be fixed in https://github.com/UniversalRobots/Universal_Robots_ROS_Driver/pull/400 and https://github.com/UniversalRobots/Universal_Robots_ROS_Driver/pull/396 (instead of using the `beta-testing` branch)

Note, that we moved the pass_through_controllers to a [new repository](https://github.com/UniversalRobots/Universal_Robots_ROS_passthrough_controllers). This one here will be archived soon (after adding a proper notification to the toplevel README. of course).
